modules/aws/master/main.tf

```diff
@@ -124,3 +124,9 @@ resource "aws_instance" "master" {
     "tectonicClusterID", "${var.cluster_id}"
   ), var.extra_tags)}"
 }
+
+resource "aws_elb_attachment" "masters" {
```

Comment:
The worker `aws_elb_attachment` resource is replacing [this][1].  But I don't see an analogous entry in master for worker nodes.  Can you explain why we need this in more detail?

[1]: https://github.com/openshift/installer/blob/fc01b3a28d5e5097336d9138b60e7e47b27da840/modules/aws/worker-asg/worker.tf#L74

Reply:
> But I don't see an analogous entry in master for worker nodes.

@yifan-gu pointed me at [the `load_balancers` entry in the old `aws_autoscaling_group.masters`][1].  That's what this entry is replacing.

[1]: https://github.com/openshift/installer/blob/fc01b3a28d5e5097336d9138b60e7e47b27da840/modules/aws/master-asg/master.tf#L21